src/templates/healthcert/types.tsx

```diff
@@ -63,3 +71,18 @@ export interface ImmunizationRecommendation {
     reference: string;
   };
 }
+
+export interface HealthCertDocument extends v2.OpenAttestationDocument {
+  name: string;
+  validFrom: string;
+  fhirVersion: string;
+  logo: string;
+  fhirBundle: {
+    resourceType: healthcert.FhirBundleResourceType.Bundle;
+    type: healthcert.FhirBundleType.Collection;
+    entry: Array<any>;
+  };
+}
+export interface NotarizedHealthCert extends HealthCertDocument {
```

Comment:
Why was the name changed to Notari**z**ed?

Reply:
i'm just use the same name like api-vaccine, is there have any other reason to keep it as `Notarised` ?

Reply:
I think we should use the British spelling as much as possible, since the frontend display is using that too, and our app repos have been renamed from `z` to `s` as well..